pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -371,10 +372,59 @@ Error: "There are at least two blocks with same name: Build Go project"
 Semaphore assigns its own unique names to nameless `blocks` items, which are
 displayed in the Semaphore 2.0 UI.
 
+### dependencies in blocks
+
+The `dependencies` property of a block is used to define the flow of execution
+between blocks within a pipeline.
+
+To specify that block with name `Block C` needs to be run once the
+blocks with names `Block A` and `Block B` finish, the `dependencies` property of
+the block with name `Block C` needs to be populated with corresponding names
+of the blocks which block with name `Block C` depends on.
+In the following example blocks with names `Block A` and `Block B` would be executed in parallel at the
```

Comment:
Can you think of an additional example? I worry that people may not understand the full power of the feature from this one example.

Reply:
The whole reference section of this property aims to explain simple mechanism upon one can build even the most complex workflows. In the last sentence, the reader is linked to the [repo with workflows examples](https://github.com/semaphoreci-demos/semaphore-demo-workflows) where the most common are specified.